templates/admin-page.php

```diff
@@ -31,7 +31,7 @@
 					<?php } ?>
 				</select>
 
-				<input type="submit" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" />
+				<input type="submit" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" id="plugin-check__submit" class="button" />
```

Comment:
Maybe make it a primary button, since it is the primary action on the page. We'll revise design later, so this is a nit-pick at this point, but a good practice.
```suggestion
				<input type="submit" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" id="plugin-check__submit" class="button button-primary" />
```